README.md

```diff
@@ -0,0 +1,186 @@
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Frameworks</summary>
+  <ul>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Responsive UI**
+- **Mobile and desktop menu**
+- **JavaScript generated section**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Visit my Live version here
+
+- [Live Demo Link](https://tomasesquivelgc.github.io/module1-capstone/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+This project is good for understanding how a website can be setup.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you just need a web broswer.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/tomasesquivelgc/module1-capstone.git
+```
+
+
+### Install
+
+No need to install anything.
+
+### Usage
+
+To run the project, simply open index.html on a web browser.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Tomás Matías Esquivel**
+
+- GitHub: [tomasesquivelgc](https://github.com/tomasesquivelgc)
+- Twitter: [tomas esquivel](https://twitter.com/EsquivelTomas)
+- LinkedIn: [tomas esquivel](https://www.linkedin.com/in/tomas-esquivel-b2160568/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- [ ] **Animations**
+- [ ] **Button functions**
+- [ ] **More sections**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/tomasesquivelgc/module1-capstone/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star and continue supporting my career!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse for helping me move forward and he following pages for providing free content for use:
+ - https://iconos8.es
+ - https://pixabay.com
+ - https://www.freepik.com
+
+And the original author of the template (Cindy Shin)[https://www.behance.net/adagio07]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Good job so far 👏 As the previous reviewer said, the Licence link is broken. Please fix that because it will help people who visit this repo to checkout the terms in the licence file 😉